db/warehouse/migrate/20210910113307_add2022v1p1_spec_items.rb

```diff
@@ -0,0 +1,12 @@
+class Add2022v1p1SpecItems < ActiveRecord::Migration[5.2]
+  def change
+    Bi::ViewMaintainer.new.remove_views
```

Comment:
You do this in both migrations, is that correct/necessary?

Reply:
Not sure how necessary it is in the second one, but it has to be in the first one and it felt safer to add it on the second essentially as a rebuild.